ui/widgets/layout/src/components/layout-widget.tsx

```diff
@@ -246,7 +246,7 @@ const Layout: React.FC<unknown> = () => {
               />
             </Stack>
 
-            <Stack customStyle="fixed bottom-2 mx-2 lg:(static my-4)">
+            <Stack customStyle="fixed bottom-2 mx-2 lg:(w-[21rem] mb-2)">
```

Comment:
Not sure if giving it a fixed width is good idea ... need to think of a better approach. Curious to see what other think here.

Reply:
Just saw the comment. I agree that giving a fixed width may not be a good idea